chapter1/OSGPProperties.md

```diff
@@ -0,0 +1,31 @@
+## I.1 Properties of the Open Smart Grid Platform
+
+- Acts as a **connecting link** between (web)applications and smart devices
+- The open Source approach **prevents vendor lock-in** 
+- State of the art **security** 
+- Fully **scalable,** dynamically scaling up and down as more devices and applications are added.  > 5 million devices with the aid of a growth path
+- Freedom of choice in the **desired IP communication infrastructure** , e.g. CDMA and GPRS.
+- Stimulates open innovation by using **open standards** and **open source** technology
+- **Multiple devices** and communication protocols are supported
+- **Independent** of (cloud) hosting infrastructure
+- By delinking the chain and the use of open standards and the open source license, everybody can build their applications on top of OSGP.
+- OSGP is **optimized** to provide reliable and efficient delivery of command and control information for e.g. [smart meters](http://en.wikipedia.org/wiki/Smart_meter), direct [load control](http://en.wikipedia.org/wiki/Load_control) modules, [solar panels](http://en.wikipedia.org/wiki/Solar_panels), gateways and other applications.
+- OSGP simplifies the implementation of smart devices resulting in a shorter time-to-market by having build-in **device management** features
+- The platform supports various IP data communication infrastructures to communicate with the devices (internet, lan, GPRS, CDMA, UMTS, etc.).
+- OSGP also supports authentication and encryption for all data exchanges to protect the integrity and privacy of data as required in e.g. the smart grid.
+- OSGP Supports **multiple protocols**
+- **Easy application integration**
+- Supports **active-active** setup over multiple datacenters
+- Adding servers can be done in **runtime**
+ 
+
+### I.1.1 Unique features of the Open Smart Grid Platform
+
+The Open Smart Grid Platform is unique by its multi-dimensional, generic and open design. Because of a true separation of layers and the use of open standards, other suppliers and/or third parties are able to develop and market innovative solutions.
+
+1. The platform is **multidimensional**. This means that several customer use cases (with separate business models) are able to use the various device functions. One single application could use the same function of different devices. 
+2. The **generic** design ensures that the platform can be used in a flexible way for several functions and applications (e.g. public lighting services and smart meter services).
+3. The platform is aimed at the **'common parts'** of the technology chain; suppliers or vendors (of both applications and devices) have no competitive advantage in delivering these kind of services.
+4. The platform layers are truly separated by open standards and the platform is made available as **open source** software.
+5. The  platform does not hold on to any application data ( **stateless** ). No messages/commands will ever get lost.
```

Comment:
Hi @kevinsmeets : The whole idea is that you will always get an answer to a request, if it can't be hadled you should receive an error message in you application. What kind of message cannot be handled and are currently passed to the DLQ?